.github/workflows/ci.yml

```diff
@@ -20,7 +20,19 @@ jobs:
       options: --privileged
     steps:
     - uses: actions/checkout@v2
-    - name: Run storage tests
+    - name: Run storage tests not marked as root as vdsm user
```

Comment:
The name is little bit too long; Maybe "Run storage tests as vdsm" and "Run storage tests as root"?